### ISSUE TITLE
Traing problem.

### ISSUE THREAD

**Issue by wqtwjt1996** [argman/EAST#284]
Here is the loss function I used:

In loss.py:
eps = 1e-5

Delete: return torch.mean(L_g * y_true_cls * training_mask) + classification_loss
Change to: return torch.sum(L_g * y_true_cls * training_mask)/(torch.nonzero(y_true_cls * training_mask).shape[0] + eps) + classification_loss

After 1000 epochs. My loss decreased from 13 to 0.0009. However, there is no bounding box from the prediction results of the model of 1000 epochs. Has anyone met the same problem? By the way, I used Win10 to run the code.
